src/app/learnocaml_student_view.ml

```diff
@@ -400,7 +401,7 @@ let display_report exo report =
     Manip.replaceChildren report_button
       Tyxml_js.Html5.[ txt [%i"Report"] ]
   end else begin
-    Manip.addClass report_button "partial" ;
+      Manip.addClass report_button "partial" ;
```

Comment:
Weird indentation. (Someday we will use ocamlformat and never think about these anymore :-) )

Reply:
Sure! But note that I had spotted it and fix this in a later commit.
So the [final patch](https://github.com/ocaml-sf/learn-ocaml/pull/548/files) (which will be squashed in a single commit) does not show this weird indentation.